js/package.json

```
@@ -35,18 +35,21 @@
     "clean": "rm -rf lib"
   },
   "peerDependencies": {
-    "@gear-js/api": "0.40.0",
-    "@polkadot/api": "15.4.1"
+    "@gear-js/api": "0.41.1",
+    "@polkadot/api": "15.9.1"
```

```suggestion
    "@gear-js/api": "^0.41.1",
    "@polkadot/api": "^15.9.1"
```

Wouldn't make sense to unpin peer deps? Feels like `sails-js`  is not getting updated each minor `@gear-js/api` update, and sometimes in apps there are conflicts that we have to resolve manually if installed versions don't match.